Cargo.toml

```diff
@@ -18,6 +18,7 @@ scripting = ["surrealdb/scripting"]
 http = ["surrealdb/http"]
 http-compression = []
 ml = ["surrealdb/ml", "surrealml-core"]
+experimental_parser = ["surrealdb/experimental_parser"]
```

Comment:
```suggestion
experimental-parser = ["surrealdb/experimental-parser"]
```

This is the naming scheme we use for our other feature flags.

Reply:
Alright, I changed the flag name!